packages/block-editor/src/components/block-toolbar/index.native.js

```diff
@@ -37,12 +37,16 @@ export default withSelect( ( select ) => {
 		getBlockMode,
 		getSelectedBlockClientIds,
 		isBlockValid,
+		getSelectedBlock,
 	} = select( 'core/block-editor' );
 	const blockClientIds = getSelectedBlockClientIds();
+	const selectedBlock = getSelectedBlock();
+	const isSelectedGroup = selectedBlock && selectedBlock.name === 'core/group';
```

Comment:
This should be using `getGroupingBlockName` instead of hardcoding the block name. Also, the name is a bit confusing and too close to `isSelected`. The [equivalent code](https://github.com/WordPress/gutenberg/blob/a97d54591900db05f050c6c64d9ed4bdf22d1a1b/packages/editor/src/components/convert-to-group-buttons/convert-button.js#L87) from the web is using `isUngroupable` instead.

Reply:
That makes sense to change that name and use `getGroupinBlockName`. 

>I think it would be interesting to explore de-duplicating the grouping and ungrouping logic and moving it to the core/block-editor store. 

Didn't dive into the logic, I've just patched missing part. If there is a need can check it and handle it in a separate issue.

Reply:
I've refactored it to follow the web approach.